tests/core/pyspec/eth2spec/test/phase0/block_processing/test_process_deposit.py

```diff
@@ -57,6 +57,9 @@ def run_deposit_processing(spec, state, deposit, validator_index, valid=True, ef
 
     assert state.eth1_deposit_index == state.eth1_data.deposit_count
 
+    if spec.fork != PHASE0:
+        assert len(state.online_countdown) == len(state.validators)
```

Comment:
Did some tests fail prior to the `add_validator` fix?

And should we write a couple sanity tests about online_countdown?

Reply:
> Did some tests fail prior to the `add_validator` fix?

Yes. It seems the `test_process_deposit.py` tests are the only place we make deposit after the genesis state, this check would fail on these test cases prior to the `add_validator` fix.

> And should we write a couple sanity tests about online_countdown?

Good call. Added some sanity checks: https://github.com/ethereum/eth2.0-specs/pull/2064/commits/df8fdd9f587fb5c8fbef6caf5ba33ed69c19e091